src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/resourcetakes/view/RecordResourceFragment.kt

```diff
@@ -1,32 +1,126 @@
 package org.wycliffeassociates.otter.jvm.app.ui.resourcetakes.view
 
-import org.wycliffeassociates.controls.ChromeableTabPane
-import org.wycliffeassociates.otter.common.data.model.ContentType
-import org.wycliffeassociates.otter.jvm.app.ui.resourcetakes.viewmodel.RecordResourceViewModel
+import com.github.thomasnield.rxkotlinfx.toObservable
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import org.wycliffeassociates.otter.jvm.app.widgets.highlightablebutton.highlightablebutton
+import javafx.beans.property.SimpleStringProperty
+import javafx.geometry.Pos
+import javafx.scene.layout.GridPane
+import javafx.scene.layout.Priority
+import javafx.scene.layout.RowConstraints
+import javafx.scene.layout.VBox
+import org.wycliffeassociates.otter.common.data.workbook.Take
+import org.wycliffeassociates.otter.jvm.app.ui.takemanagement.view.RecordableFragment
+import org.wycliffeassociates.otter.jvm.app.ui.takemanagement.viewmodel.RecordableViewModel
+import org.wycliffeassociates.otter.jvm.app.widgets.takecard.TakeCard
+import org.wycliffeassociates.otter.jvm.app.widgets.takecard.resourcetakecard
 import tornadofx.*
-import org.wycliffeassociates.otter.jvm.utils.getNotNull
 
-class RecordResourceFragment : Fragment() {
-    private val viewModel: RecordResourceViewModel by inject()
-    private val tabPane = ChromeableTabPane()
+class RecordResourceFragment(recordableViewModel: RecordableViewModel) : RecordableFragment(recordableViewModel) {
+    val formattedTextProperty = SimpleStringProperty()
 
-    override val root = tabPane
+    private val newTakeButton =
+        highlightablebutton {
+            highlightColor = Color.ORANGE
+            secondaryColor = AppTheme.colors.white
+            isHighlighted = true
+            graphic = MaterialIconView(MaterialIcon.MIC_NONE, "25px")
+            maxWidth = 500.0
+            text = messages["newTake"]
+            action {
+                recordableViewModel.recordNewTake()
+            }
+        }
 
-    // The tabs will add or remove themselves from the tabPane when their view model's 'recordable' property changes
-    @Suppress("unused")
-    private val tabs: List<RecordableTab> = listOf(
-        recordableTab(ContentType.TITLE, 0),
-        recordableTab(ContentType.BODY, 1)
-    )
+    private val leftRegion = VBox().apply {
+        vgrow = Priority.ALWAYS
 
-    private fun recordableTab(contentType: ContentType, sort: Int) = RecordableTab(
-        viewModel.contentTypeToViewModelMap.getNotNull(contentType),
-        tabPane,
-        sort,
-        viewModel::onTabSelect
-    )
+        //selected take and drag target
+        stackpane {
+            // drag target glow
+            add(dragComponents
```

Comment:
code formatting on all of these add methods

Reply:
All of these add methods are going away since I'm making a new drag target component anyway